ikabot/function/sellResources.py

```diff
@@ -17,43 +17,81 @@
 from ikabot.helpers.process import set_child_mode
 from ikabot.helpers.planearViajes import waitForArrival
 
-t = gettext.translation('sellResources', 
-                        localedir, 
-                        languages=idiomas,
+t = gettext.translation('sellResources',
+                        localedir,
+                        languages=languages,
                         fallback=True)
 _ = t.gettext
 
-def chooseComertialCity(ciudades_comerciales):
+def chooseCommercialCity(commercial_cities):
+	"""
+	Parameters
+	----------
+	commercial_cities : list[dict]
+
+	Returns
+	-------
+	commercial_city : dict
+	"""
 	print(_('In which city do you want to sell resources?\n'))
-	for i, ciudad in enumerate(ciudades_comerciales):
-		print('({:d}) {}'.format(i + 1, ciudad['name']))
-	ind = read(min=1, max=len(ciudades_comerciales))
-	return ciudades_comerciales[ind - 1]
-
-def getStoreInfo(s, ciudad):
-	params = {'view': 'branchOfficeOwnOffers', 'activeTab': 'tab_branchOfficeOwnOffers', 'cityId': ciudad['id'], 'position': ciudad['pos'], 'backgroundView': 'city', 'currentCityId': ciudad['id'], 'templateView': 'branchOfficeOwnOffers', 'currentTab': 'tab_branchOfficeOwnOffers', 'actionRequest': 'REQUESTID', 'ajax': '1'}
-	resp = s.post(params=params, noIndex=True)
+	for i, city in enumerate(commercial_cities):
+		print('({:d}) {}'.format(i + 1, city['name']))
+	ind = read(min=1, max=len(commercial_cities))
+	return commercial_cities[ind - 1]
+
+def getStoreInfo(session, city):
```

Comment:
maybe beter to name it `market` instead of `store`

Reply:
yes, totally